api.md

```diff
@@ -950,6 +1029,79 @@ SELECT device_id, time_bucket('5 minutes', time) as interval,
  alternative to an `ORDER BY time DESC LIMIT 1` clause to find the
  latest value (which will use indexes).
 
+## locf() [](locf)
+
+The `locf` function (last observation carried forward) allows you to carry the last seen value in an aggregation group forward.
+It can only be used in an aggregation query with [time_bucket_gapfill](#time_bucket_gapfill).
+The `locf` function call cannot be nested inside other function calls.
+
+#### Required Arguments [](locf-required-arguments)
+
+|Name|Description|
+|---|---|
+| `value` | The value to carry forward (anyelement) |
+
+#### Optional Arguments [](locf-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `prev` | The lookup expression for values before gapfill start (anyelement) |
+
+The `prev` expression will only be evaluated when no previous value is returned by the outer query.
+
```

Comment:
Can we also add that little snippet here: This happens when the first bucket in the queried time range is empty or does not otherwise contain a valid value?

Reply:
Agree.  Or perhaps just copy explanation or reference back to longer explanation given in interpolation?

Reply:
Same nit:

The `prev` expression will only be evaluated when no previous value is returned by the outer query (i.e., the first bucket in the queried time range is empty).